File: retos-python/16_limpiando_nieve.py
#  Resultado 5 ⭐
import re

def remove_snow(s: str)-> str:
    regex = r"(.)\1"

    while re.search(regex, s):
        matches = re.findall(regex, s)
        for match in matches:
            s = re.sub(f"{match}{match}", "", s)

    return s

File: retos-ts/16_limpiando_nieve.ts
//  Resultado 5 ⭐
function removeSnow(s: string): string {
    const regex = /(.)\1/g;
  
    while(regex.test(s)){
      const matches: string[] | null = s.match(regex)
      if(matches){
          matches.forEach(match => {
              s = s.replace(match,'')
          })
      }
    }
  
    return s;
}